scripts/genmap.sh

```diff
@@ -1,7 +1,19 @@
 #!/usr/bin/env bash
```

Comment:
genmap.sh is now a generic, independent tool for adding map generation plugin to custom worlds with custom configuration